### ISSUE TITLE
[Feature Request] Optimize UI for mobile devices

### ISSUE THREAD

**Issue by arlcrow** [victoralvesf/aonsoku#98]
Tauri supports building applications for Android, but the current UI is not optimized for mobile devices. To enhance user experience and accessibility, the interface should be adjusted for smaller screens and touch interactions.

**Comment by arlcrow** [comment]
![Screenshot_20241214-144022](https://github.com/user-attachments/assets/a13f0988-0ebb-4bee-be1a-f18d958d1341)
![Screenshot_20241214-144037](https://github.com/user-attachments/assets/1d2ac445-0aee-4f8c-846e-e8bf91fa50b2)
![Screenshot_20241214-144042](https://github.com/user-attachments/assets/8629d19b-f202-43a0-99db-002dfe75fafa)

In landscape mode the app looks almost usable.

![Screenshot_20241214-144048](https://github.com/user-attachments/assets/bb699cf5-13fb-463b-b294-58e7f16ffd10)
![Screenshot_20241214-144058](https://github.com/user-attachments/assets/3449a62b-3178-4eda-92cd-c65b13eb9094)

In portrait mode it doesn't.

**Comment by victoralvesf** [comment]
Hi @arlcrow, thank you for your suggestion! However, this app is currently designed for desktop use only. In version `0.7.1`, I’ll add a message for mobile devices and hide the UI to avoid confusion.

In the future, I might consider creating a separate project specifically for Android and iOS.
